frontend/src/metabase/query_builder/components/dataref/ModelPane/ModelPane.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { connect } from "react-redux";
+import { t, jt } from "ttag";
+import { createSelector } from "reselect";
+import PropTypes from "prop-types";
+
+import { getMetadata } from "metabase/selectors/metadata";
+import MetabaseSettings from "metabase/lib/settings";
+import { formatDateTimeWithUnit } from "metabase/lib/formatting";
+import {
+  Description,
+  EmptyDescription,
+} from "metabase/components/MetadataInfo/MetadataInfo.styled";
+import Question from "metabase-lib/lib/Question";
+import FieldList from "../FieldList";
+import {
+  ModelPaneDetail,
+  ModelPaneDetailLink,
+  ModelPaneDetailText,
+  ModelPaneIcon,
+  ModelPaneDescription,
+} from "./ModelPane.styled";
+
+// This formats a timestamp as a date using any custom formatting options.
+function formatDate(value) {
```

Comment:
If possible, use `<DateTime />` component to render a date. It will handle custom formatting settings. Otherwise use something like this: https://github.com/metabase/metabase/blob/e2a765fc804f60849844f399b86223b06264431b/frontend/src/metabase/components/DateTime.jsx#L39